pulsar-common/src/main/java/com/yahoo/pulsar/common/api/Commands.java

```diff
@@ -534,15 +534,19 @@ public static ByteBuf newConsumerStatsResponse(CommandConsumerStatsResponse.Buil
     private final static ByteBuf cmdPing = serializeWithSize(
             BaseCommand.newBuilder().setType(Type.PING).setPing(CommandPing.getDefaultInstance()));
 
-    static ByteBuf newPing() {
-        return RecyclableDuplicateByteBuf.create(cmdPing).retain();
+    public static ByteBuf newPing() {
+        ByteBuf pingCmdBuf =  RecyclableDuplicateByteBuf.create(cmdPing).retain();
```

Comment:
Thats what I was wondering. With one static instance of cmdPing, why do we need pooled buffer. Can we just "return cmdPing"?